stock_picking_invoice_link/migrations/11.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    sql = """
```

Comment:
There's an openupgradelib method for avoiding you to write this SQL: https://github.com/OCA/openupgradelib/blob/884331200591e098ed1a0963934660bcef8075c0/openupgradelib/openupgrade.py#L1235